config/stored_requests.go

```diff
@@ -330,8 +330,14 @@ func (inMemCache *InMemoryCache) validate() error {
 	case "none":
 		return nil
 	case "unbounded":
-		if inMemCache.TTL <= 0 {
-			return errors.New("Stored requests In-Memory caches need a TTL when unbounded")
+		if inMemCache.TTL != 0 {
```

Comment:
So ... unbounded memory caches need to have to TTL and no max size, so we cache everything forever until the OOM killer gets to it?

Should be `if inMemCache.TTL == 0`

Reply:
Yes, that's exactly how it works today :). It's just implemented using a `sync.Map`, which doesn't support any of those options

Reply:
Ahh, ok. Nevermind then.